hack/generator/pkg/conversions/property_conversion_context.go

```diff
@@ -3,67 +3,80 @@
  * Licensed under the MIT license.
  */
 
-package astmodel
+package conversions
 
-// StorageConversionContext captures additional supporting information that may be needed when a
+import "github.com/Azure/azure-service-operator/hack/generator/pkg/astmodel"
+
+// PropertyConversionContext captures additional supporting information that may be needed when a
 // storage conversion factory creates a conversion
-type StorageConversionContext struct {
+type PropertyConversionContext struct {
 	// types is a map of all known type definitions, used to resolve TypeNames to actual types
-	types Types
+	types astmodel.Types
 	// functionName is the name of the function we're currently generating
 	functionName string
 	// knownLocals is a reference to our set of local variables
 	// (Pointer because it's a reference type, not because it's optional)
-	knownLocals *KnownLocalsSet
+	knownLocals *astmodel.KnownLocalsSet
 }
 
-// NewStorageConversionContext creates a new instance of a StorageConversionContext
-func NewStorageConversionContext(types Types, idFactory IdentifierFactory) *StorageConversionContext {
-	return &StorageConversionContext{
+// NewStorageConversionContext creates a new instance of a PropertyConversionContext
+func NewStorageConversionContext(types astmodel.Types, idFactory astmodel.IdentifierFactory) *PropertyConversionContext {
+	return &PropertyConversionContext{
 		types:       types,
-		knownLocals: NewKnownLocalsSet(idFactory),
+		knownLocals: astmodel.NewKnownLocalsSet(idFactory),
 	}
 }
 
+//TODO: Remove this?
```

Comment:
So should you? :wink:

Reply:
Those methods were introduced as a part of relocating things one by one into the new package, but I'll keep them. Fixed the comments.